Library/ViewModels/SearchProjectCellViewModel.swift

```diff
@@ -10,11 +10,16 @@ public protocol SearchProjectCellViewModelInputs {
 public protocol SearchProjectCellViewModelOutputs {
   var deadlineSubtitleLabelText: Signal<String, NoError> { get }
   var deadlineTitleLabelText: Signal<String, NoError> { get }
-  var fundingLargeLabelText: Signal<NSAttributedString, NoError> { get }
   var fundingSmallLabelText: Signal<NSAttributedString, NoError> { get }
-  var projectImageUrlMed: Signal<URL?, NoError> { get }
+  var fundingLargeLabelText: Signal<NSAttributedString, NoError> { get }
   var projectImageUrlFull: Signal<URL?, NoError> { get }
-  var projectNameLabelText: Signal<String, NoError> { get }
+  var projectImageUrlMed: Signal<URL?, NoError> { get }
+  var projectName: Signal<String, NoError> { get }
+  var projectNameLabelText: Signal<NSAttributedString, NoError> { get }
+  var progress: Signal<Float, NoError> { get }
+  var progressBarColor: Signal<UIColor, NoError> { get }
+  var percentFundedText: Signal<NSAttributedString, NoError> { get }
+  var metadataText: Signal<String, NoError> { get }
```

Comment:
I'm using this in the `MostPopularSearchProjectCell`. I can rename this to `MostPopularSearchProjectCellViewModel`.